alf/utils/value_ops.py

```diff
@@ -82,8 +85,8 @@ def action_importance_ratio(action_distribution, collect_action_distribution,
             importance_ratio, 1 - importance_ratio_clipping,
             1 + importance_ratio_clipping)
     elif clipping_mode == 'capping':
-        importance_ratio_clipped = tf.clip_by_value(importance_ratio, 0,
-                                                    importance_ratio_clipping)
+        importance_ratio_clipped = tf.clip_by_value(
```

Comment:
tf.minimum(importance_ratio, 1 + importance_ratio_clipping)
This is more efficient.

Reply:
done.